polaris-core/src/main/java/org/apache/polaris/core/config/PolarisConfiguration.java

```diff
@@ -162,13 +162,8 @@ public Builder<T> catalogConfig(String catalogConfig) {
       return this;
     }
 
-    /**
-     * Used to support backwards compatability before there were reserved properties. Usage of this
-     * method should be removed over time.
-     */
-    @Deprecated
```

Comment:
I know the alerts are annoying, but I think we need to get in the habit of marking things we want to remove as `Deprecated` and would like to keep the annotation

Reply:
That's not to say we can't change our minds about this btw, just noting this is by design. Maybe bad design :)

Reply:
Annoying to whom? ;) Right now they seems to be very visible to use users, who cannot do anything to "fix" it :shrug:

Reply:
Only to users building from source. Otherwise, it's meant to be visible to developers -- a reminder & incentive to deprecate the method, not one to just yank the `@Deprecated` annotation.

Reply:
Looks good! Just want to flag that there is a surviving log [here](https://github.com/apache/polaris/blob/0a484bb32c867178cb2ddac24d1280bab8b34ab6/polaris-core/src/main/java/org/apache/polaris/core/config/PolarisConfigurationStore.java#L132), but IMO we should keep that one because it tells users to stop using the config, rather than developers to stop building Polaris with a method call in place.

Reply:
The "surviving" log is quite appropriate IMHO. It informs the user that _they_ ought to update their table's properties to use the new name.